cluster/addons/registry/images/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM haproxy:1.5
```

Comment:
Yes, the http header will contain localhost/127.0.0.1 when forwarded with socat so the registry will (sensibly) error.